src/test/java/michael/StorageTest.java

```diff
@@ -0,0 +1,21 @@
+package michael;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StorageTest {
+    @Test
+    public void Test1() {
+        Storage s = new Storage("./data/save.txt");
+        Event e = new Event("Test", "Mon 2pm", "4pm");
+        assertEquals("E | 0 | Test | Mon 2pm | 4pm\n", s.convertTaskToString(e));
+    }
+
+    @Test
+    public void Test2() {
```

Comment:
The test name should comply to "featureUnderTest_testScenario_expectedBehavior()".